utils/common/src/main/kotlin/Os.kt

```diff
@@ -118,6 +118,13 @@ object Os {
         }
     }
 
+    /**
+     * Prepend [path] to the system's PATH environment.
+     */
+    fun prependToPathEnvironment(path: File) {
+        env["PATH"] = path.toString() + env["PATH"]?.let { File.pathSeparatorChar + it }
```

Comment:
Won't this append a literal "null" string if "PATH" was unset before?

Also, should we maybe add a sanity check to only prepend if `path` is not already the first item?

Reply:
> Won't this append a literal "null" string if "PATH" was unset before?

Unfortunately, yes.

> 
> Also, should we maybe add a sanity check to only prepend if `path` is not already the first item?

Good point. 

How about this ?

```
    fun prependToPathEnvironment(path: File) {
        val paths = env["PATH"]?.split(File.pathSeparatorChar)
        val absolutePath = path.absolutePath
        if (paths != null) {
            if (absolutePath !in paths) {
                env["PATH"] = absolutePath + File.pathSeparatorChar + env["PATH"]
            }
        } else {
            env["PATH"] = absolutePath
        }
    }
```